grakn-engine/src/main/java/ai/grakn/engine/user/UsersHandler.java

```diff
@@ -31,6 +33,7 @@
  * @author Marco Scoppetta
  */
 public class UsersHandler {
```

Comment:
Should we merge `UsersHandler` and `SystemKeyspaceUsers`. Is there a case in which we will use `UsersHandler` on its own?